scout/adapter/mongo/variant_loader.py

```diff
@@ -664,7 +664,7 @@ def load_variants(
             if vcf_dict["category"] != category:
                 continue
 
-            LOG.debug("Attempt to load %s %s VCF.", variant_type, category.upper())
+            LOG.info(f"\nLoading'{vcf_file_key}' variants")
```

Comment:
Moved the LOG.info in the variant loader, so it prints the message for any variant that is loaded, clinical or research